tools/vagrant/clean_all_vagrand_vms.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+IDsVMs=($(VBoxManage list vms | awk -F '[{}]' '{print $2}'))
+
+for idvm in "${IDsVMs[@]}";
```

Comment:
I don't think this is a good idea since the user can have more machines running not related with the project.

What is the difference from running `vagrant destroy`.

Reply:
```vagrant destroy --force``` will destroy all vms in the vagrant instance created by the current directory.
```make vagrand-clean-all-vms``` will destroy all vms `IN ALL INSTANCES` created by any directory.

Yes, it's kind of `risk` but might help users that have multiple instance of `sig-windows-dev` running unintentional and generating network conflict as we noticed in my lab. :/

Reply:
Maybe a warning: `Are you sure you want to remove ALL VMs created in the SYSTEM?` and ask users to type "yes" to continue?

Reply:
Is there a way to tag a VM? If yes, then can we tag them as `sig-windows-dev-tools` and only delete VMs with that tag? Otherwise we are just adding an endpoint to delete all VMs on your host and I am worried folks are going to be caught by surprise even with the promot.